Files.Launcher/Program.cs

```diff
@@ -589,19 +590,48 @@ private static async Task ParseFileOperationAsync(AppServiceRequestReceivedEvent
                     });
                     break;
 
-                case "DragDrop":
-                    cancellation.Cancel();
-                    cancellation.Dispose();
-                    cancellation = new CancellationTokenSource();
-                    var dropPath = (string)args.Request.Message["droppath"];
-                    var dropText = (string)args.Request.Message["droptext"];
-                    var drops = Win32API.StartSTATask<List<string>>(() =>
-                    {
-                        var form = new DragDropForm(dropPath, dropText, cancellation.Token);
-                        System.Windows.Forms.Application.Run(form);
-                        return form.DropTargets;
-                    });
-                    break;
```

Comment:
Was this removed on purpose?

Reply:
@yaichenbaum That's an issue while resolving merge conflicts. I have fixed it and restored the code